tools/distribution/BUILD

```diff
@@ -9,31 +10,26 @@ exports_files([
     "distrotest.sh",
 ])
 
-envoy_py_library(
-    name = "tools.distribution.distrotest",
-    data = [
-        "distrotest.yaml",
-    ],
+envoy_py_command(
+    name = "tools.distribution.release",
+    command_main = "envoy.distribution.release",
```

Comment:
im wondering if `entry_point` is a better name for this, i didnt call it that mostly because entry_points arent just for `console_scripts` - maybe we should call it that

Reply:
Yes, please call it `entry_point`. And also rename to `envoy_py_binary` with this new support.